src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/resource/ResourceEditor.test.tsx

```diff
@@ -170,4 +170,75 @@ describe("ResourceEditor", () => {
     userEvent.click(actionCenter);
     expect(screen.getByTestId("elements-copy")).not.toBeInTheDocument;
   });
+
+  it("opens AddElementDialog, interacts with it, and can close it", async () => {
+    const setInitialFormikValuesStu6 = jest.fn();
+    const setValidationSchema = jest.fn();
```

Comment:
Just to be consistent with other mocked functions, We could name these fn starting with mock ?